projects/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls        import path
+
+from .views             import MainListView
+
+urlpatterns = [
+    path("", MainListView.as_view()),
```

Comment:
Main이라는 용어는 메인페이지에 필요한 엔드포인트로 생각하셨기 때문에 사용하신 것 같네요!
API는 페이지를 기준으로 만들면 안되고, 자원(데이터, 예를들면 상품, 유저, ...)을 기준으로 작성해주셔야 합니다. 

그래야 엔드포인트를 여러 곳에서 호출해서 사용할 수 있습니다~

```suggestion
    path("", ProductListView.as_view()),
```

Reply:
넵 앞으로 리소스 기준으로 뷰이름 작성하겠습니다